packages/xo-web/src/common/intl/messages.js

```diff
@@ -416,7 +417,6 @@ const messages = {
   deleteBackupScheduleQuestion:
     'Are you sure you want to delete this backup job?',
   deleteSelectedJobs: 'Delete selected jobs',
-  scheduleEnableAfterCreation: 'Enable immediately after creation',
```

Comment:
Yes, we lose a lot of translations but this can confuse the user and we already have the title "New Schedule" which is always displayed and we add this message. We can add a condition to differentiate edition/creation, What do you think?